UMI3D-Browser-Desktop/Assets/Dependencies/UMI3D SDK/ClientDevelopmentKit/Collaboration/Runtime/Networking/UMI3DCollaborationClientServer.cs

```diff
@@ -384,7 +386,9 @@ public static void SendVOIP(int length, byte[] sample)
         protected override async Task<byte[]> _GetFile(string url, bool useParameterInsteadOfHeader)
         {
             UMI3DLogger.Log($"GetFile {url}", scope);
-            return await (environmentClient?.GetFile(url, useParameterInsteadOfHeader) ?? Task.FromResult<byte[]>(null));
+            if (environmentClient != null)
```

Comment:
This can be simplify. your are testing two time if EnvironmentClient is null. Please implement a getfile method in the worldcontrolerclient to mirror the environmentclient.

Reply:
Get file can still be null...